cms/search/signals.py

```diff
@@ -0,0 +1,46 @@
+from django.apps import apps
```

Comment:
This should be `signal_handlers.py` for consistency

Reply:
Done!